ts/src/base/Exchange.ts

```diff
@@ -3742,6 +3742,69 @@ export default class Exchange {
         return [ false, params ];
     }
 
+    async handlePoTif (accountMarketType: string, isMarketOrder: boolean, params: any = {}) {
+        // our unified strings
+        const unifiedTifKey = 'timeInForce';
+        const unifiedTifKey_IOC = 'IOC';
+        const unifiedTifKey_FOK = 'FOK';
+        const unifiedTifKey_PO = 'PO';
+        // const unifiedTifKey_GTC = 'GTC';
+        // get exchange TIF mappings
+        const tifOptionsAllMarkets = this.safeValue (this.options, 'timeInForceMap', {});
+        // accountMarketType might 'spot', 'swap', 'unified', 'whatever_custom' ...
+        const tifOptions = this.safeValue (tifOptionsAllMarkets, accountMarketType, {});
+        // get exchangeSpecific TIF key
+        const exchangeSpecificTifKey = this.safeString (tifOptions, 'key');
+        // get exchangeSpecific TIF values map
+        const exchangeSpecificTifMap = this.safeValue (tifOptions, 'strings', {});
+        // unified or exchangeSpecific TIF value (if set in PARAMS)
+        const providedTifValue = this.safeString2 (params, exchangeSpecificTifKey, unifiedTifKey);
+        // exchangeSpecific values for different TIME IN FORCE values
+        const exTifValue_IOC = this.safeString (exchangeSpecificTifMap, unifiedTifKey_IOC);
+        const exTifValue_FOK = this.safeString (exchangeSpecificTifMap, unifiedTifKey_FOK);
+        const exTifValue_PO = this.safeString (exchangeSpecificTifMap, unifiedTifKey_PO);
+        // const exTifValue_GTC = this.safeString (exchangeSpecificTifMap, unifiedTifKey_GTC);
+        //
+        // #################### TIF CHECK #################### //
+        //
+        // check if unified 'timeInForce' (if set in PARAMS) matches to either unified or exchange-specific value
+        const is_TIF_IOC = this.inArray (providedTifValue, [ unifiedTifKey_IOC, exTifValue_IOC ]);
+        const is_TIF_FOK = this.inArray (providedTifValue, [ unifiedTifKey_FOK, exTifValue_FOK ]);
+        const is_TIF_PO = this.inArray (providedTifValue, [ unifiedTifKey_PO, exTifValue_PO ]);
+        // const is_TIF_GTC = this.inArray (providedTifValue, [ unifiedTifKey_GTC, exTifValue_GTC ]);
+        // #################### BOOLEAN CHECK #################### //
```

Comment:
few things are commented, impropertly formated and non-standard Variable names - these are just temporary, to bring more readability for you for quick understanding.